torchgeo/trainers/simclr.py

```diff
@@ -176,13 +166,16 @@ def configure_models(self) -> None:
 
         # Create projection head
         input_dim = self.backbone.num_features
-        if hidden_dim is None:
-            hidden_dim = input_dim
-        if output_dim is None:
-            output_dim = input_dim
+        if self.hparams["hidden_dim"] is None:
```

Comment:
Just for my understanding, all these changes are so you can do this?

Reply:
Yes